### DISCUSSION TITLE
EVE-NG M1 problem

### DISCUSSION THREAD

**Discussion by nOuTySqVorA** [utmapp/UTM#6592] [UTM Help (macOS)]
I have a problem with connection to my virtual network devices(such like Mikrotik, Cisco etc) via telnet.  I downloaded EVE-NG client for my MacOS(M1 Pro) from here: https://www.eve-ng.net/index.php/download/, installed it in UTM and tried to connect to virtual network devices via telnet but there was only one output anytime:

````
Trying 192.168.87.12...
Connected to 192.168.87.12.
Escape character is '^]'.
````
After that, nothing happening. I have found this article https://www.reddit.com/r/networking/comments/satzou/eveng_on_macos_monterey_with_telnet/ and tried the following steps:

* installed Telnet via Homebrew and used its path in ITerm2 profile like */opt/homebrew/bin/telnet $$HOST$$ $$PORT$$* - without success 
* tried to use /usr/local/bin/telnet - the same result
* tried to set path via launchctl: launchctl config user path "/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin"

Also I have tried to enable debug mode for telnet connection - without success. I would be glad if some knows how to fix this problem or could give me an advise. Thanks:)

**Comment by plenartowicz** [comment]
The same issue on M2 and MacOS Sonoma. 
I've tried the same solutions and nothing helped.

But I'm more and more convinced that it is problem with MacOS itself.
I have not tried it yet https://www.eve-ng.net/forum/viewtopic.php?f=3&t=4980 but maybe it could help?
